src/nbt/tag.rs

```diff
@@ -74,28 +77,21 @@ impl NbtTag {
         bytes.freeze()
     }
 
-    /// Serializes as single NBT tag without name.
-    pub fn serialize_tag(&self) -> Bytes {
+    /// Writes tag id and its data into bytes
```

Comment:
Before comment was used to show clear difference between serialize, and serialize_named. Now it is unnecessary.

Reply:
Someone might be not sure what's the difference beteen serialize and serialize_raw (and deserialize respectively), hence the comment.

Reply:
1. Change order it should be `serialize`, then `serialize_raw`.
2. Describe only `serialize_raw`
3. Why to use `write` word instead of `serialize`?

Reply:
1 and 2 - sure
3 - it seemed nicer when I was working on this, but serialize is good too, I'll change it